src/transport/CASESession.cpp

```diff
@@ -35,11 +35,32 @@
 #include <support/CodeUtils.h>
 #include <support/ReturnMacros.h>
 #include <support/SafeInt.h>
+#include <support/BufferWriter.h>
 #include <transport/SecureSessionMgr.h>
 
 namespace chip {
 
+// Dummy x509 certificate...
+// TODO: Check whether this should be kept or not
```

Comment:
I agree with your comment. We should not keep it here. Is it being used as a test vector? If so, we can move it to the tests folder.